admin-perspective/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
```

Comment:
Does someone without cluster admin rights will only see alerts that are associated with namespaces that they have read access to?

Reply:
In the admin console at least, I believe that all the things that fall under "monitoring" are cluster scoped and only visible to admins. I was under the impression that this initial work would be scoped the same way, but I think a similar pattern could be useful for project-specific alerts when they exist as well